experiments/dice_experiment/run_dice_experiment.py

```diff
@@ -0,0 +1,553 @@
+"""A mainfile for running DICE experiments.
+
+It executes a batch of DICE runs. If the --experiment flag is provided, it
+constructs the batch of run configs from an experiment config by performing
+grid search over the experiment config parameters.
+
+If the --distributed flag is provided, it uses parallel_runner.py to
+split the runs into batches and execute them in parallel. The number of
+parallel processes is given by --num_processes."""
+
+# TODO(@jakeval): Reduce duplication. https://github.com/jakeval/MRMC/issues/44
+
+import os
+import sys
+import pathlib
+
+#  Append MRMC/. to the path to fix imports.
+mrmc_path = pathlib.Path(os.path.abspath(__file__)).parent.parent.parent
+sys.path.append(str(mrmc_path))
+
+from typing import Optional, Mapping, Sequence, Tuple, Any
+import shutil
+import pathlib
+import json
+import argparse
+
+from data import data_loader
+from data.datasets import base_loader
+from data.adapters import continuous_adapter
+from data import recourse_adapter
+from models import model_constants
+from models import model_interface
+from models import model_loader
+from recourse_methods import dice_method
+from core import recourse_iterator
+from core import utils
+from experiments import utils as experiment_utils
+from experiments import parallel_runner
+
```

Comment:
Nit: Only need one empty line here and below. Two blank lines is to appear at the top of a non-class function.